test/fluree/db/json_ld/api_test.cljc

```diff
@@ -213,17 +213,17 @@
                loaded-db    (fluree/db loaded)]
            (is (= target-t (:t loaded-db)))
            (testing "query returns expected `list` values"
-             (is (= #{{:id         :ex/cam,
-                       :type       :ex/User,
-                       :ex/numList [7 8 9 10]}
-                      {:id :ex/john, :type :ex/User}
-                      {:id         :ex/alice,
-                       :type       :ex/User,
-                       :ex/friends [{:id :ex/john} {:id :ex/cam}]}}
-                    (set @(fluree/query loaded-db {:context [test-utils/default-context
-                                                             {:ex "http://example.org/ns/"}]
-                                                   :select  '{?s [:*]}
-                                                   :where   '{:id ?s, :type :ex/User}})))))))
+             (is (= [{:id         :ex/alice,
```

Comment:
Deterministic query result ordering looks to be back, which is nice. But is it deterministic? I'm not sure that namespace code generation will always happen in the same order. Though I guess most of our tests only use a single user namespace, so maybe it's fine here.

Reply:
Namespace generation will always happen in the order the namespaces are encountered. We aren't randomizing the test input, so the outputs should always be the same.